### ISSUE TITLE
Link error on older Android devices

### ISSUE THREAD

**Issue by GregCornis** [ARM-software/armnn#757]
I'm trying to use the AAR in an Android App, as presented in the [AI Talk](https://www.youtube.com/watch?v=Zu4v0nqq2FA). The version with the GPU delegate works fine, but when I add the dependency on `armnn_delegate_jni-23.11.aar` and add an `ArmnnDelegate()` to the config, I get a link error:
```
java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "strtod_l" referenced by "/data/app/com.*****/lib/arm64/libarmnn_delegate_jni.so"...
  	at java.lang.Runtime.loadLibrary0(Runtime.java:989)
      	at java.lang.System.loadLibrary(System.java:1562)
 	at com.arm.armnn.delegate.ArmnnDelegate.<init>(ArmnnDelegate.kt:21)
	at com.cornis.core.image_processing.TFLiteSegmenter.<init>(TFLiteSegmenter.kt:24)
```

The device I am running on is on Android API 25, with a Mali T860. 

I've found a related issue upstream, though I am not 100% sure what the link is: https://github.com/tensorflow/tensorflow/issues/61951.

**Comment by GregCornis** [comment]
Unfortunatly I cannot upgrade the device. From what I understood of tensorflow's issue, they mean to revert the `NDK_API_LEVEL` to 21 for next release, so I hope that I can use `armnn` on API 25. Is that explicitly not supported on your side ?
Do you know where the error is coming from ?

**Comment by nikraj01** [comment]
Hello @GregCornis ,

Looking at the code found out that we were building the AAR file targeting API level 27. That's why you were getting the error in your device which has API level of 25. I have got the build with the API of your device. I will let you know from where you can pick up the binary to test it.

Thanks,
Nikhil.

**Comment by GregCornis** [comment]
Awesome thanks! I'm available for any test that you want me to do

**Comment by GregCornis** [comment]
Hi @nikraj01, any news on this issue ? Thanks :)

**Comment by nikraj01** [comment]
Hi @GregCornis ,

Unfortunately I am waiting for some approval before I send it to you. I really appreciate your patience.

Thanks,
Nikhil.

**Comment by nikraj01** [comment]
Hello @GregCornis ,

Thanks very much for raising the issue and your patience. As its not an actual issue in Arm NN and is related to an older version of Android in your particular device, which is bit old and not something we have on our side to test it we won't be able to provide the AAR file. We don't want to provide a file to our users which is untested. We understand it should have been documented somewhere about the minimum API version support. For that reason we have now updated our release note for our latest release with the minimum API level we have tested and supported. Hope that clarifies the issue.

Thanks,
Nikhil.
